src/storage/mod.rs

```diff
@@ -686,6 +689,124 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
         }
     }
 
+    pub fn scan_lock(
```

Comment:
Do we have test case for the latter arrived async commit transaction and checking its `commitTS` with safe point ts?